EVENT.md

```diff
@@ -980,6 +980,10 @@ were reported.
 
 In order to achieve the privacy goals listed above the API has various rate limits, which can be found [here](https://github.com/WICG/attribution-reporting-api/blob/main/params/chromium-params.md) for Chromium.
 
+Note: Browsers may choose to allow separate rate limits for the embedded site 
```

Comment:
Is there a reason the explainer uses `embedded site` but the spec uses `embedded origin`?

Reply:
That’s just because the explainer mentions “top-level site” in that section, whereas the spec change is where we get the origin.